changelogs/fragments/9892-vmadm-add-optional-options.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - vmadm - add optional options (https://github.com/ansible-collections/community.general/pull/9892).
```

Comment:
```suggestion
  - vmadm - add new optional options ``flexible_disk_size`` and ``owner_uuid`` (https://github.com/ansible-collections/community.general/pull/9892).
```

Reply:
And a minor nit: "optional options" sounds... redundant. Maybe "optional parameters" ? Same idea, sounding slightly less odd (IMHO, at least)?

Reply:
I changed "optional options" to "optional parameters" and added the ``flexible_disk_size`` and ``owner_uuid`` as well. I can commit the changes when I get an answer on the other comments.

Reply:
How about simply removing the "optional"? I adjusted my suggestion accordingly.